project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/components/c4m-app/src/directives/types/types.html

```diff
@@ -1,3 +1,3 @@
-<div class="form-group btn-group btn-group-{{field}}">
-  <button type="button" ng-value="value.id" ng-class="{active: value.id==type}" ng-model="type" ng-required="!discussionType" ng-click="updateType(value.id, field)" class="btn btn-default {{field}} col-xs-12 col-lg-{{cols}}" ng-repeat="value in allowedValues"><i class="fa"> {{ value.label }}</button>
+<div class="form-group btn-group-{{field}}">
+  <button type="button" ng-value="value.id" ng-class="{active: value.id==type}" ng-model="type" ng-required="!discussionType" ng-click="updateType(value.id, field)" class="btn btn-default {{field}} col-xs-12 col-lg-{{cols}}" ng-repeat="value in allowedValues"><i class="fa"></i> {{ value.label }}</button>
```

Comment:
No, it's being used for the icons targeting [here](https://github.com/capacity4dev/capacity4more/blob/dd21fe6dcb0d719681acba01f35cfdf05e38b6df/project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_quickpost.scss#L91-L112).